src/renderers/common/Renderer.js

```diff
@@ -1916,8 +1916,6 @@ class Renderer {
 			renderContext.renderTarget = renderTarget;
 			renderContext.depth = renderTarget.depthBuffer;
 			renderContext.stencil = renderTarget.stencilBuffer;
-			// #30329
-			renderContext.clearColorValue = this._clearColor;
```

Comment:
With the new code this line can be removed which is good since we have overlooked that `renderContext.clearColorValue` must be in working color space and must premultiply alpha. Both were missing so far.